lib/chai/core/assertions.js

```diff
@@ -2015,7 +2018,7 @@ module.exports = function (chai, _) {
   /**
    * ### .decrease(function)
```

Comment:
The same applies for this line, what about renaming `function` to `target` on these cases?